homeworks/anatoliy.pogoriliy_AyzekUorren/homework_7/task1.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generates numbers in the Fibonacci sequence.
+ */
+class Fibonacci {
+    /**
+     * @param {number} n
+     * @returns The 'n'th number in the Fibonacci sequence.
+     */
+    generate(n) {
+        if (n < 2) {
+            return n;
+        }
+
+        return this.generate(n - 1) + this.generate(n - 2);
+    }
+}
+
+module.exports = Fibonacci;
```

Comment:
don't see your function/class method returns an array...

also, it could be more matching the task to accept the maximum value as argument, and not the number of elements

Reply:
I created two ways, generate to number of element and generate to max.